### ISSUE TITLE
Improve loading of files in file browser

### ISSUE THREAD

**Issue by cloudhead** [radicle-dev/radicle-upstream#561]
Currently files "pop" into existence after the folder is opened. We should use some fancy loading animation.

**Comment by juliendonck** [comment]
Is there no way to make it faster?

**Comment by cloudhead** [comment]
Yeah, there is, but there will always be a slow computer or a large repo on which things will be slow..

For the B1 though, it could be quicker to improve performance than to add loading animations.

**Comment by juliendonck** [comment]
yeah was thinking the same

**Comment by rudolfs** [comment]
As of https://github.com/radicle-dev/radicle-upstream/pull/620 the entire tree is loaded in one go:

![tree](https://user-images.githubusercontent.com/158411/86374355-c11f9980-bc84-11ea-80f6-444ea412d792.gif)

Can we close this issue now?

**Comment by xla** [comment]
Let's reopne a new one with an in-depth outline of the expectations we have for this component.
